docs/create_new_resource.md

```diff
@@ -0,0 +1,164 @@
+# Create a new resource
+
+As an example, let's create a Book entity:
+
+<!-- TOC -->
+* [Create the entity](#create-the-entity)
+* [Configure the BookRepository](#configure-the-bookrepository)
+<!-- TOC -->
+
+## Create the entity
+
+To create a new entity, we need to run the following command:
+
+```shell
+$ bin/console make:entity 'App\Entity\Book'
+```
+
+This command is interactive: it will guide you through the process of adding all the fields you need.
+Use the following answers (most of them are the defaults, so you can hit the "Enter" key to use them):
+
+|   Name | Author | Description | Price |
+|-------:|--------|:-----------:|:------|
+| String | String |    Text     | Float |
+|    255 | 255    |      /      |  /    |
+|     No | No     |     No      | No    |
+
+****
+
+Here is the full output when running the command:
+
+```
+created: src/Entity/Book.php
+created: src/Repository/BookRepository.php
+
+Entity generated! Now let's add some fields!
+You can always add more fields later manually or by re-running this command.
+
+New property name (press <return> to stop adding fields):
+> name
+
+Field type (enter ? to see all types) [string]:
+> 
+
+Field length [255]:
+>
+
+Can this field be null in the database (nullable) (yes/no) [no]:
+> 
+
+updated: src/Entity/Book.php
+
+Add another property? Enter the property name (or press <return> to stop adding fields):
+> author
+
+Field type (enter ? to see all types) [string]:
+> 
+
+Field length [255]:
+>
+
+Can this field be null in the database (nullable) (yes/no) [no]:
+> 
+
+updated: src/Entity/Book.php
+
+Add another property? Enter the property name (or press <return> to stop adding fields):
+> description
+
+Field type (enter ? to see all types) [string]:
+> text
+
+Can this field be null in the database (nullable) (yes/no) [no]:
+> 
+
+updated: src/Entity/Book.php
+
+Add another property? Enter the property name (or press <return> to stop adding fields):
+> price
+
+Field type (enter ? to see all types) [string]:
+> float
+
+Can this field be null in the database (nullable) (yes/no) [no]:
+> 
+
+updated: src/Entity/Book.php
+
+Add another property? Enter the property name (or press <return> to stop adding fields):
+> 
+
+
+       
+Success! 
+       
+
+Next: When you're ready, create a migration with php bin/console make:migration
+
+```
+
```

Comment:
We should show the result of these commands + we are missing resource tag adding

Reply:
Also, don't we need to add some configuration to make it work?